readyplayerme/Unity-Core
# Settings and Module Updates

### Changes
- Settings file structure changed to help other packages use it
- Module update logic cleaned up
- Package json updated

### How to test
- Import the core package from https://github.com/readyplayerme/Unity-Core.git#feature/webview-updates
- Make a release for WebView and increase the version number
- Click Ready Player Me > Check for Updates button
- Expect package to be updated
